e2e/nx/src/run.test.ts

```diff
@@ -768,6 +768,11 @@ describe('Nx Running Tests', () => {
           scripts: {
             build: 'nx exec -- echo HELLO',
             'build:option': 'nx exec -- echo HELLO WITH OPTION',
+            compound: 'nx exec -- "echo HELLO && echo COMPOUND"',
+            'compound:option':
+              'nx exec -- "echo HELLO && echo COMPOUND" && echo OPTION',
```

Comment:
I don't understand what this is supposed to be testing, `&& echo OPTION` will always be run in the root context.

Reply:
i removed this one, but added one for shell and one for node